app/views/step_by_step_pages/schedule.html.erb

```diff
@@ -0,0 +1,82 @@
+<%
+  links = [
+    {
+      text: 'Step by step publisher',
+      href: step_by_step_pages_path
+    },
+    {
+      text: @step_by_step_page.title,
+      href: step_by_step_page_path(@step_by_step_page)
+    },
+    {
+      text: 'Schedule'
+    }
+  ]
+%>
+
+<%= render 'shared/steps/step_breadcrumb', links: links %>
+
+<%= render 'shared/steps/heading',
+  step_nav: @step_by_step_page.title,
+  action: 'Set a date and time to publish'
+%>
+
+<%= render 'nav', step_by_step_page: @step_by_step_page, active: "schedule" %>
+
+<% if @schedule_intent %>
+  <%= render 'shared/steps/form_errors', resource: @schedule_intent %>
+<% end %>
+
+<% if @step_by_step_page.status[:name] == 'live' %>
```

Comment:
f the status is live, couldn't the whole page be hidden instead?

Reply:
What do you mean by hidden? (We can hide the 'Schedule' button from the publish-or-delete.html, but what happens if a user with Scheduling permissions gets to this page anyway?)

Reply:
We hide the publish button if a draft doesn't exist:  https://github.com/alphagov/collections-publisher/blob/master/app/views/step_by_step_pages/publish_or_delete.html.erb#L28

We could nest the schedule button inside the same check.

Reply:
It's unlikely that a content designer here is going to try and hack the url and cause mischief, so I'm not worried about adding extra checks. It would be different if we were opening step by step publishing to the wider publishing community.